conf/clusterrunner-master

```diff
@@ -0,0 +1,99 @@
+#!/bin/sh
+### BEGIN INIT INFO
```

Comment:
I usually add `set -euxo pipefail` at the top of any non-trivial bash script to make bash behave more sanely
https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/

Reply:
Did you see my original comment. I didn't write this file. It's from https://github.com/fhd/init-script-template.

If we want to make change we probably should try to do them upstream.

Reply:
Ah, didn't realize that this file was copy pasted. What do you mean upstream? The only thing upstream from this file is systemd right? It's not like we can change that. And even if we could I don't think any bug that would be avoided by `set -euxo pipefail` could be properly handled by anything other than the bash script itself (downstream or upstream).

I _can_ see the aversion to modifying a copy-pasted file. Afaikt this file looks bug free right now and `set -euxo pipefail` won't really do anything. My main worry is that we'll make modifications to this file, introduce some bug and that bug will be exacerbated by bash's shitty default behavior. My secondary worry is that there's already a bug in this file that I haven't spotted that will be exacerbated by bash's shitty default behavior.